instrumentation/jms/src/main/java/brave/jms/PropertyFilter.java

```diff
@@ -76,7 +76,7 @@ static void filterProperties(Message message, Set<String> namesToClear, List<Obj
       String name = out.get(i).toString();
       try {
         message.setObjectProperty(name, out.get(i + 1));
-      } catch (JMSException e) {
+      } catch (Throwable e) {
```

Comment:
we tend to use Call.propagatedIfFatal when catching throwable.

Reply:
Fixed in next line.